styles/custom/variable.scss

```diff
@@ -1,4 +1,5 @@
 $padding: 20px;
+$paddingVW: 2vw;
```

Comment:
I never used vw before I must say. Maybe you can help me here, why you did choose to use it :)